src/validation.cpp

```diff
@@ -1875,8 +1876,8 @@ static unsigned int GetBlockScriptFlags(const CBlockIndex* pindex, const Consens
     // mainnet and testnet), so for simplicity, always leave P2SH
     // on except for the one violating block.
     if (consensusparams.BIP16Exception.IsNull() || // no bip16 exception on this chain
-        pindex->phashBlock == nullptr || // this is a new candidate block, eg from TestBlockValidity()
-        *pindex->phashBlock != consensusparams.BIP16Exception) // this block isn't the historical exception
+        pindex->m_hash_block.IsNull() || // this is a new candidate block, eg from TestBlockValidity()
```

Comment:
is this kosher 100% of the time as a replacement for nullptr?

Reply:
This should fix itself in the next rebase?